schema-engine/sql-schema-describer/src/postgres.rs

```diff
@@ -1122,6 +1150,79 @@ impl<'a> SqlSchemaDescriber<'a> {
         Ok(())
     }
 
+    /// Return the constraints that are not primary keys, foreign keys, or unique keys.
+    /// Namely, this returns CHECK and EXCLUDE constraints.
+    async fn get_constraints(
+        &self,
+        sql_schema: &mut SqlSchema,
+    ) -> DescriberResult<IndexMap<(String, String), BitFlags<TableProperties, u8>>> {
+        let namespaces = &sql_schema.namespaces;
+        let sql = include_str!("postgres/constraints_query.sql");
+        let rows = self
+            .conn
+            .query_raw(
+                sql,
+                &[Array(Some(namespaces.iter().map(|v| v.as_str().into()).collect()))],
+            )
+            .await?;
+
+        let mut constraints_map: IndexMap<(String, String), BitFlags<TableProperties, u8>> = IndexMap::default();
+
+        let upsert_bitflag = |constraints_map: &mut IndexMap<(String, String), BitFlags<TableProperties, u8>>,
```

Comment:
We don't need a lambda for this. It's really hard to read, and you can just replace it with much simpler check (if you just use a standard BTreeMap):

```rust
constraints_map.entry(&constraint_key).and_modify(|curr| curr |= flag).or_insert(flag)
```

Also having closure parameters by-value, such as `(String, String)` is not really a good idea. Remvove the whole thing, and just use the `entry/and_modify/or_insert` trick in the two places where you call this.